package.json

```diff
@@ -91,6 +91,7 @@
         "yarn-deduplicate": "^1.1.1"
     },
     "dependencies": {
+        "@emotion/is-prop-valid": "^0.8.2",
```

Comment:
@Gerigot 
Could you please clarify whether duplicating this from `optionalDependencies` was necessary?

Reply:
this is probably a mistake and should be removed

Reply:
Oh I see now I'm sorry yes it was a mistake probably done automatically by my IDE, if it's a problem I'll remove it.

Reply:
I'm not using `preact` in projects w/ framer-motion and was just passing by :)
Since you probably have a working state w/ React 17, could you please test whether it still installs w/o this dep in `dependencies` and remove afterwards?

Reply:
I'm an idiot. 
I was "fixing" (removing) the dependency and I screw the things up with git. so I've created a new pull request with the fix.

Reply:
I'm confused. I have a Next.js project that uses React 17. When I do `npm i framer-motion` I still get the peer dependency error. I thought the problem was fixed? Is there something I need to do to get the latest, fixed version of your library to install? I forced the install with `npm i framer-motion --force` but was hoping this wouldn't be necessary...thanks!

Reply:
I have the same exact problem as [Amasa-Delano](https://github.com/framer/motion/pull/847#discussion_r531385531) .

Reply:
I see that this pull request has been merged in the main branch 11 days ago, but no version of the library has been created since then.
The problem should be that  the last version on NPM doesn't have this fix. 
You'll have to wait till a version will be created or for now download and build it by yourself.
I'm not the owner of the project, so I don't know if is planned to create a new version and when will that happen.

Reply:
thanks for your quick response and help.